client/modules/i18n/startup.js

```diff
@@ -117,7 +122,7 @@ Meteor.startup(() => {
     const shopLanguage = (shop && shop.language) || null;
 
     // Use fallbacks to determine the final language
-    const language = userLanguage || shopLanguage || "en";
+    const language = getCookieValue("lngStorefrontGreenstorm") ||  userLanguage || shopLanguage || "de";
```

Comment:
I think we need to keep `en` as the default language for the app.

Reply:
Guys sorry, we thought we closed this, because it was open by mistake. Will close it right now. Sorry again.